<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formLoading</class>
 <widget class="QDialog" name="formLoading">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QFrame" name="frameLoad">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>99</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame { background-color: rgb(231, 231, 231);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="laSystemName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>28</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 28pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>QCPF Framework</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>110</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>110</y>
       <width>110</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labPwd">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOk">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>110</y>
       <width>110</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUser">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>14</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPwd">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>63</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labUsr">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labInfoPrefix_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>424</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(104, 104, 104);</string>
   </property>
   <property name="text">
    <string>Copyright © Jamie.T</string>
   </property>
  </widget>
  <widget class="QLabel" name="labCoreVersion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>4</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(104, 104, 104);</string>
   </property>
   <property name="text">
    <string>Core Verson:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labViewVersion">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>425</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(104, 104, 104);</string>
   </property>
   <property name="text">
    <string>View Verson:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBackgroud">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/bin/Images/gbLoad.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_LoadingInfo">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>375</y>
     <width>430</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>labelBackgroud</zorder>
  <zorder>frameLoad</zorder>
  <zorder>labInfoPrefix_2</zorder>
  <zorder>labCoreVersion</zorder>
  <zorder>labViewVersion</zorder>
  <zorder>lb_LoadingInfo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
